### ISSUE TITLE
New item notifications gone

### ISSUE THREAD

**Issue by jsv94** [FMaz008/VineHelper#112]
After the most recent update and the change in frequency in checking for new items from 1 minute to 30 seconds, i no longer receive any notifications for new items. I've removed and re-added the plugin, restarted my chrome and computer, none of my site settings have changed... i dont know what to try next.

**Comment by FMaz008** [comment]
The code as not changed at all for this feature.
Are you referring to the toaster notification of the notification manager ?

**Comment by Rainydaymitchell** [comment]
the new item notification is gone for me too.

**Comment by Rainydaymitchell** [comment]
> The code as not changed at all for this feature. Are you referring to the toaster notification of the notification manager ?

not sure what this means.

**Comment by FMaz008** [comment]
Sorry I had a typo in my question. Let me rephrase:
Are you using the notification monitor, or the on screen notifications?

If you are using the notitication monitor, does it show ON (green icon)?

**Comment by jsv94** [comment]
It's the on screen notifications. They DO show up on the notifications monitoring page but they no longer show up on the vine page screen. I attached screenshots of what i have selected on the plugin and what the notifications monitor page shows. 

<img width="1044" alt="Screenshot 2024-05-25 at 21 24 11" src="https://github.com/FMaz008/VineHelper/assets/170700922/bc954b10-38a9-471d-b6f7-d3f289554d4e">
<img width="1037" alt="Screenshot 2024-05-25 at 21 24 31" src="https://github.com/FMaz008/VineHelper/assets/170700922/61e1154a-4aea-4d37-8e65-76adab649be0">

**Comment by Rainydaymitchell** [comment]
The on screen pop up on the vine page no longer shows for me either. I did remove the extension and re-added it. This did not help. There is a group of 9 of us that are having the same issue. The sound and the pop-up does not show. In order to see new items we must go to the Notifications monitoring page and keep that page open. The box to have the pop up keeps getting unchecked and we all don't know why.  We have all decided to just remove the vine helper if we can't figure it out by this coming week.

**Comment by FMaz008** [comment]
> We have all decided to just remove the vine helper if we can't figure it out by this coming week.

What do you expect from this kind of behavior? You realize I do this for free, right?

**Comment by jsv94** [comment]
I'm the OP. I just wanted to bring it to your attention. I am extremely appreciative of what you've created and have no intention of removing the plugin myself regardless of this small issue. Thank you for everything!!!!

**Comment by FMaz008** [comment]
> I'm the OP. I just wanted to bring it to your attention. I am extremely appreciative of what you've created and have no intention of removing the plugin myself regardless of this small issue. Thank you for everything!!!!

Yeah, this has been confirmed by a few people already, and I'll look into it as soon as possible.

**Comment by FMaz008** [comment]
The issue was identified and a fix was implemented. New version should be release in 8-10 days.

**Comment by s2daxlgn** [comment]
In fact @Rainydaymitchell go ahead and post proof of the donation you will be making today... We would all love to see it.

**Comment by Rainydaymitchell** [comment]
Omg. I truly did not want to offend you at all. My deepest apologies. You have created a wonderful app and it helps so many people. We work in the office and the pop ups let us sneak vine on our CS computers. By saying we may need to remove it, that  had no ill intent at all. I now realize that this extension was created by a small intelligent group of people and not a big corp. Again my apologies for creating this hostile chat. May God bless all of you and your hard work. 

Yahoo Mail: Search, Organize, Conquer 
 
  On Sun, May 26, 2024 at 8:59 AM, Francois ***@***.***> wrote:   



We have all decided to just remove the vine helper if we can't figure it out by this coming week.


What do you expect from this kind of behavior? You realize I do this for free, right?

**Comment by Rainydaymitchell** [comment]
As for the donation, let me match what you've donated so you can go ahead and post that too. We all would love to see it.

Yahoo Mail: Search, Organize, Conquer 
 
  On Thu, May 30, 2024 at 2:07 AM, ***@***.***> wrote:   


In fact @Rainydaymitchell go ahead and post proof of the donation you will be making today... We would all love to see it.

**Comment by s2daxlgn** [comment]
> As for the donation, let me match what you've donated so you can go ahead and post that too. We all would love to see it. Yahoo Mail: Search, Organize, Conquer On Thu, May 30, 2024 at 2:07 AM, ***@***.***> wrote: In fact @Rainydaymitchell go ahead and post proof of the donation you will be making today... We would all love to see it. — Reply to this email directly, view it on GitHub, or unsubscribe. You are receiving this because you were mentioned.Message ID: ***@***.***>

I HAVE donated thank you very much. Stop being a douchebag and if you are using the extension - give the guy a donation instead of being a demanding tool.

**Comment by Rainydaymitchell** [comment]
I already said that I would be happy to donate and I would love to match what you have donated. Together. We could be a team and we could donate. Just show me what you donated and I will match it.

**Comment by s2daxlgn** [comment]
> I already said that I would be happy to donate and I would love to match what you have donated. Together. We could be a team and we could donate. Just show me what you donated and I will match it.

CAN YOU READ?? I already donated. You don't need to see what I donated to donate for the extension YOU are using and DEMANDING things of. Boy are you really a first class prick. AND you are supposed to be working but spending all day on Vine. Tells you about your lack of character. This is my last response. Stop being a sleazy weasel.

**Comment by Rainydaymitchell** [comment]
Sure no problem. God bless you. Have a wonderful day. Thank you so much for all you do to help out.

Yahoo Mail: Search, Organize, Conquer 
 
  On Thu, May 30, 2024 at 12:59 PM, ***@***.***> wrote:   



I already said that I would be happy to donate and I would love to match what you have donated. Together. We could be a team and we could donate. Just show me what you donated and I will match it.


CAN YOU READ?? I already donated. You don't need to see what I donated to donate for the extension YOU are using and DEMANDING things of. Boy are you really a first class prick. AND you are supposed to be working but spending all day on Vine. Tells you about your lack of character. This is my last response. Stop being a sleazy weasel.
